cli/src/common/boto3/ec2.py

```diff
@@ -30,11 +30,9 @@ def describe_instance_type_offerings(self):
 
     @AWSExceptionHandler.handle_client_exception
     def describe_ami_id_offering(self, ami_id):
-        """Return a boolean to determine ami id is valid or not."""
-        return self._paginate_results(
-            self._client.describe_images(
-                ImageIds=[
-                    ami_id,
-                ]
-            )
+        """Return a dict of ami info."""
```

Comment:
Thanks for fixing this method.

1. let's rename the method to describe_image
2. let's return the dictionary or raise an AWSClientError(..., "image xxx not found") if the ami is not found, then the caller will manage the exception.

Reply:
done, thanks for clear example